spec/shared_print/finder_spec.rb

```diff
@@ -59,6 +71,25 @@
       # Get nothing if there are no phase 3.
       expect(described_class.new(organization: [org1], phase: [p3]).commitments.to_a).to eq []
     end
+    it "is specific" do
+      # i.e. when you query for phase 3 commitments,
+      # you get only commitments with phase 3
+      cluster_tap_save(
+        build(:commitment, ocn: ocn1, phase: 1),
+        build(:commitment, ocn: ocn1, phase: 2),
+        build(:commitment, ocn: ocn1, phase: 3)
+      )
+      # Look up commitments using a phase array.
+      # Expect the phases of all commitments in the output
+      # to match the phases of the input.
+      phase_in_phase_out([1])
```

Comment:
I think I would be tempted to make an uber-array `[[1], [2], [3], ...]` and then do an `.each` on it to run the individual subtests. Seems a bit more DRY-ish.